scripts/oneoff/get_g12_recovery_services.py

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env python
+"""
+Get the drafts that correspond to G12 recovery services. Note that there can be 0 or multiple drafts for one service.
+You will need to process the output to get a one-to-one mapping.
+
+Usage:
+    scripts/get_g12_recovery_services.py <stage> <phase_file> <output_file>
+"""
+import csv
+import sys
+
+from dmapiclient import DataAPIClient
+from dmutils.env_helpers import get_api_endpoint_from_stage
+from docopt import docopt
+
+sys.path.insert(0, ".")
+
+from dmscripts.helpers.auth_helpers import get_auth_token
+
+
+def _normalise_service_name(name):
```

Comment:
You might need more rules, underscore for example.

Reply:
I'd copied from another script that was less comprehensive. I've now updated the normalisation to follow the script you linked. As a result, it found a few new matching drafts.